src/main/java/com/microsoft/graph/content/BatchRequest.java

```diff
@@ -1,85 +0,0 @@
-// ------------------------------------------------------------------------------
```

Comment:
I don't believe we should be dropping batching

Reply:
they're dropped in the meantime while I go ahead and refactor them to work with the new request adapter pattern hence me only commenting out it's implementation in the baseClient class since I know they still will belong there in some capacity.

Reply:
I don't think @andrueastman had to remove them during the transition for dotnet. I believe there was a brief moment when core was aligned with kiota,  tasks were present but not functional.  Andrew could you share some insights on how you managed that to help Ramses please? I'd rather not delete them to keep history etc. Maybe we can temporary move them instead of exclude them with a project setting?

Reply:
Sure, we can designate a temp package/folder for them and just exclude references of them for now.